src/resources/2024/ChestMimic.ts

```diff
@@ -136,17 +136,14 @@ export function differentiate(
 /**
  * Check how many of a monster is available to differentiate into
  *
- * @param monster The monster to differentiate your egg into; may behave weirdly with name collisions
+ * @param monster The monster to differentiate your egg into
  * @returns How many of a Monster we can differentiate
  */
 export function differentiableQuantity(monster: Monster): number {
   if (!have_($item`mimic egg`)) return 0;
-  const regex = new RegExp(
-    `<!-- monsterid: ${monster.id}-->(?:\\s\\((\\d+)\\))?`
-  );
+  const regex = new RegExp(`<!-- monsterid:${monster.id} -->\\s\\((\\d+)\\)`);
```

Comment:
```suggestion
  const regex = new RegExp(`<!-- monsterid:${monster.id} --> \\((\\d+)\\)`);
```

No need in mixing whitespace literals and whitespace metacharacters before/after `-->` unless you expect it to dynamically change.